opennms-webapp/src/main/webapp/about/index.jsp

```diff
@@ -70,7 +70,7 @@
 <table class="table table-sm">
   <tr>
     <th>Version:</th>
-    <td><a href="https://docs.opennms.com/horizon/<%=Vault.getProperty("version.display")%>/releasenotes/whatsnew.html" target="_blank" title="Release Notes"><%=Vault.getProperty("version.display")%></a></td>
+    <td><a href="https://docs.opennms.com/<%=Vault.getProperty("opennms.product")%>/<%=Vault.getProperty("release.version")%>/releasenotes/whatsnew.html" target="_blank" title="Release Notes"><%=Vault.getProperty("opennms.product")%> - <%=Vault.getProperty("release.version")%></a></td>
```

Comment:
Don't add a new property "release.version", use "version.display" instead, docs page path and properties values would be adjusted accordingly. @cgorantla @indigo423 please correct me if am wrong. 
```suggestion 
    <td><a href="https://docs.opennms.com/<%=Vault.getProperty("opennms.product")%>/<%=Vault.getProperty("release.version")%>/releasenotes/whatsnew.html" target="_blank" title="Release Notes"><%=Vault.getProperty("opennms.product")%> - <%=Vault.getProperty("version.display")%></a></td>
```